### ISSUE TITLE
Filter UI for upcoming meetings page handle 'GEN'  area incorrectly

### ISSUE THREAD

**Issue by jennifer-richards** [ietf-tools/datatracker#4824]
### Describe the issue

The upcoming meetings page shows a column with heading 'GEN', corresponding to the General Area group. When clicked, this toggles the state of the Systers and Tools buttons and sets `?show=gen` in the URL. However, the Systers and Tools group meetings are not actually shown in this state.

The GEN column should not be shown at all because its groups are in the "Other" column.

![Screenshot from 2022-12-02 16-23-05](https://user-images.githubusercontent.com/19472766/205379683-007ca0df-13ff-4b0e-9328-7f1409e0ab9a.png)

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by rjsparks** [comment]
But it would show if there were any GEN area groups with future interims (such as gendispatch or shmoo).

**Comment by jennifer-richards** [comment]
Right. It shouldn't be shown right now, though, because there are no such groups with meetings to show/hide.
